### PR TITLE
Refactor StaticShapeLoopLayerTest

### PR DESCRIPTION
**Pull request by olpipi**
### Details:
 - Refactor StaticShapeLoopLayerTest

### Tickets:
 - [CVS-111364](https://jira.devtools.intel.com/browse/CVS-111364)

### CODE REVIEW

**Comment by olpipi** [comment]
https://openvino-ci.toolbox.iotg.sclab.intel.com/job/private-ci/job/ie/job/build-linux-macos_arm64/5755/

File: src/tests/functional/shared_test_classes/src/single_op/loop.cpp [hunk 1]
```diff
@@ -121,5 +125,132 @@ void LoopLayerTest::SetUp() {
...
+    bool unrolling;
+    bool static_iter_num;
+    bool static_continue_cond;
+    int64_t max_iter_num;
+    int64_t dynamic_exit;
+    int64_t axis;
+    int64_t start_value;
+    ov::Shape data_shape;
+    ov::element::Type model_type;
+    auto args_papck = std::tie(static_iter_num, max_iter_num, dynamic_exit, axis);
```
Comment (vurusovs) [inline src/tests/functional/shared_test_classes/src/single_op/loop.cpp:178]: args_papck -> args_pack

File: src/tests/functional/shared_test_classes/src/single_op/loop.cpp [hunk 2]
```diff
@@ -4,11 +4,15 @@
 
 #include "shared_test_classes/single_op/loop.hpp"
 
-#include "transformations/control_flow/unroll_tensor_iterator.hpp"
 #include "openvino/op/parameter.hpp"
 #include "openvino/op/constant.hpp"
 #include "openvino/op/result.hpp"
+#include "openvino/op/concat.hpp"
```
Comment (vurusovs) [inline src/tests/functional/shared_test_classes/src/single_op/loop.cpp:10]: is it used?

**Review by vurusovs — APPROVED** [review]
Please, fix comments in separate PR

**Pull request merged**